### PR TITLE
testTag1 can report intercept failed

### LINKED ISSUE #14552: Correct intermittant test failure in com.ibm.ws.jsp23.fat.tests.JSPCdiTest.testTag1
The error 'injection failed' arises in com.ibm.ws.jsp23.fat.tests.JSPCdiTest.testTag1 when testTag2 runs before testTag1.  They share the same check on Pojo1.counter and testTag1 as it is currently written assumes it runs first, but this is not guaranteed.

### PR DESCRIPTION
**Pull request by epj**
fixes #14552

### CODE REVIEW

**Comment by epj** [comment]
#build

**Review by wtlucy — APPROVED** [review]
the copyright year should be updated, but other than that looks good!

**Comment by epj** [comment]
#libbybot

File: dev/com.ibm.ws.jsp.2.3_fat/test-applications/TestInjection.war/src/com/ibm/ws/jsp23/fat/testinjection/tagHandler/JspCdiHitMeTag.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2015 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsp.2.3_fat/test-applications/TestInjection.war/src/com/ibm/ws/jsp23/fat/testinjection/tagHandler/JspCdiHitMeTag.java:2]: This should be a date range `2015, 2020` please get this fixed in a new PR associated with the same issue. Thanks!

**Pull request merged**